src/expressionDescriptor.ts

```diff
@@ -556,15 +557,15 @@ export class ExpressionDescriptor {
       }
     }
 
-    let minute = minuteExpression;
-    let second: string = "";
+    const minute = minuteExpression;
+    let second: string = '';
```

Comment:
I'd like to stick with double quotes for formatting.  I use [Prettier](https://prettier.io/) for formatting.

Reply:
Sorry, didn't notice it. Fixed it